ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -42,6 +42,16 @@ impl KeyId {
     pub const fn item_id(&self) -> u8 {
         self.0 as u8
     }
+
+    pub fn extend_to<T>(&self, buf: &mut T) -> usize
```

Comment:
To be honest I don't really understand this implementation, why do we require the buffer to have the same length as what we extend it with?

Reply:
Looking at the code, this seems to be a mistake. Maybe the intention was to limit the number of keys the user can request. The F9R documentation specifies that `This message is limited to containing a maximum of 64 key IDs.` If I were to guess, maybe the intention was to limit the buffer to `MAX_CFG_KEYS = 64`. That is something @stevenlovegrove could confirm/deny.

Reply:
Anyway, in the meantime I've decided to remove the `asser` alltogether.